sys/sys_amd64.const

```diff
@@ -56,7 +56,7 @@ DN_RENAME = 16
 EFD_CLOEXEC = 524288
 EFD_NONBLOCK = 2048
 EFD_SEMAPHORE = 1
-EPOLLET = 2147483648
+EPOLLET = 18446744071562067968
```

Comment:
Can it be that your gcc assumes -m32 by default? And I don't think that we add -m64. That could potentially explain some sign extension weirdness.